src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -90,6 +96,8 @@ public void requestWithoutHeaderPassed() throws Exception {
 
     /**
      * LastModifiedCachingWire can cache GET requests.
+     * @todo: #120 inline redundant "headers" variable
```

Comment:
@westonized can we make a puzzle to inline all redundant variables in this test, not only `headers`?